packages/matchbox/src/components/Box/tests/Box.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Box from '../Box';
+import 'jest-styled-components';
+
+describe('Box', () => {
+  it('it should render correctly', () => {
+    const wrapper = global.renderStyled(
+      <Box
+        py="400"
+        color="gray.100"
+        bg={['blue.200', 'blue.500', 'blue.800']}
+        borderRadius="pill"
+        border="400"
+        fontSize="200"
+        lineHeight="200"
+        fontFamily="monospace"
+      >
+        Just a Box
+      </Box>
+    );
+    expect(wrapper).toMatchSnapshot();
```

Comment:
This snapshot will get out of date easily with these values - it might be worth mocking out the theme config values so that doesn't happen?